docs/as-module.rst

```diff
@@ -82,6 +82,9 @@ metadata of a Profile. :class:`Profile` instances can be created with:
 - :meth:`Profile.get_followers`
    Profiles that follow given user.
 
+- :meth:`Profile.get_followed_hashtags`
+   Hashtags that given user has followed.
+
```

Comment:
This list starts with

> With the `Profile` class, Instaloader also makes it easy to access metadata of a Profile. `Profile` instances can be created with:

However, `Profile.get_followed_hashtags()` returns `Hashtag` instances. It should not be listed in this list.

Reply:
Removed it, since it will included in `structures.rst`